test/support/http.ex

```diff
@@ -0,0 +1,12 @@
+defmodule HttpSupport do
+  def prepare_response(url) do
```

Comment:
Do we need that “prepare” here? The “body” function only describes the result with the word “body”, so the response function can be also named just “response”, doesn’t it?

Reply:
No we dont fixed, thanks